include/baresip.h

```diff
@@ -894,7 +894,7 @@ struct ua   *uag_find(const struct pl *cuser);
 struct ua   *uag_find_msg(const struct sip_msg *msg);
 struct ua   *uag_find_aor(const char *aor);
 struct ua   *uag_find_param(const char *name, const char *val);
-struct ua   *uag_find_requri(const char *requri);
+struct ua   *uag_find_requri(const struct pl *requri);
```

Comment:
this breaks the API. any way to avoid this ?

Reply:
We have `pl` from parsing the `carg->prm` with `re_regex()`. The change in the API makes sense in order to reduce the code lines. E.g.:
```diff
-       err = pl_strdup(&uri, &pluri);
-       if (err)
-               goto out;
-
        if (!ua)
-               ua = uag_find_requri(uri);
+               ua = uag_find_requri(&pluri);
```